README.md

```diff
@@ -1,697 +1 @@
-    case 3:
-      ...
-  }
-
-  switch (f) {
-    case 1:
-    case 2:
-      return doForBoth()
-  }
-  ```
-
-## Whitespace
-
-  - No irregular whitespace[no-irregular-whitespace]
-
-  ```js
-  // BAD
-  function test (  test) {
-    return 'test'
-  }
-  ```
-
-  - There should be no spacing on array brackets except for deconstructing array[array-bracket-spacing]
-
-  ```js
-  // BAD
-  const a = [ 'test' ]
-  const b = [ 'foo', 'arf']
-  var [x,y,x] = a
-
-  // GOOD
-  const a = ['boo']
-  const c = ['foo', 'bar']
-  var [ x, y, z ] = a
-  ```
-
-  - Spacing on commas should be after the commas[comma-spacing]
-
-  ```js
-  // BAD
-  var arr = [ 1 ,2 ]
-  var c = [ 1 , 2 ]
-
-  // GOOD
-  var b = [ 2, 3 ]
-  ```
-
-  - Indention on blocks should be 2 spaces[indent]
-
-  ```js
-  // BAD
-  function fn () {
-        wee()
-  }
-
-  // GOOD
-  function test () {
-    yey()
-  }
-  ```
-
-  - Spacing on object keys should be after the colon and not before[key-spacing]
-
-  ```js
-  // BAD
-  const obj = { foo :5 }
-  const a = { poke : 'er' }
-
-  // GOOD
-  const obj = { foo: 'bar' }
-  ```
-
-  - Spacing on keywords should be before and after[keyword-spacing]
-
-  ```js
-  // BAD
-  if(test) { ... }
-  function fn() { ... }
-  if (true){ ... }
-
-  // GOOD
-  if (test) { ... }
-  function fn () { ... }
-  ```
-
-  - No multiple empty lines[no-multiple-empty-lines]
-
-  ```js
-  // BAD (MAXIMUM 2)
-  const a = 5
-
-
-
-  cosnt b = 3
-
-  // OKAY
-  const a = 5
-
-
-  const b = 3
-
-  // GOOD
-  const a = 5
-
-  const b = 4
-  ```
-
-  - There should be a space after open curly brace and before close curly brace for objects and destructor[object-curly-spacing]
-
-  ```js
-  // BAD
-  const obj = {foo: 'bar'}
-  const a = {foo: ['test'] }
-  const b = ({foo}) => { ... }
-
-  // GOOD
-  const a = { foo: 'bar' }
-  const b = { foo: ['test']}
-  const c = ({ foo }) => { ... }
-  ```
-
-  - There should be a space before and after a block[space-before-blocks]
-
-  ```js
-  // BAD
-  if (a){
-    c()
-  }else {
-    t()
-  }
-
-  function a(){ .. }
-
-  // GOOD
-  if (a) {
-    d()
-  } else {
-    g()
-  }
-
-  function a() { ... }
-  ```
-
-  - There should be space before and after a function parenthesis[space-before-function-paren]
-
-  ```js
-  // BAD
-  function a() { ... }
-  class Foo {
-    constructor() {
-      // DO SOMETHING
-    }
-  }
-
-  // GOOD
-  function a () { ... }
-  class Foo {
-    constructor () { ... }
-  }
-  ```
-
-  - There should a space before and after an arrow - ES6[arrow-spacing]
-
-  ```js
-  // BAD
-  const a = ()=>{}
-
-  // GOOD
-  const a = () => {}
-  ```
-
-## Code Structure
-
-  - No unreachable code[no-unreachable]
-
-  ```js
-  // BAD
-  function fn () {
-    x = 1
-    return x
-    x = y
-  }
-  ```
-
-  - All require statements should be arranged*[no-mixed-requires]
-
-  ```js
-  // FOR THIS RULE THERE IS ALSO AN OPTION TO ARRANGE THE REQUIRE STARTING FROM CORE, MODULE, FILE, etc. Should we enable this?
```

Comment:
Hmm. So we separate core module and file? It will be like this:
```js
const fs = require('fs')

const _ = require('lodash')

const model = require('./model')
```
Does that sound good to you lot?